### ISSUE TITLE
require(*.css) in imported component

### ISSUE THREAD

**Issue by tx44** [researchgate/gemini-react#2]
Hi, @SevInf, and thanx for this plugin.

I'm trying to integrate `gemini` with one ui library based on `React` and stuck with some issues. One I corrected in this [PR](https://github.com/researchgate/gemini-react/pull/1). Another question is about styles requirement inside imported components in gemini tests. How to deal with this?

I created example repo for this case: https://github.com/teryaew/gemini-react-example
Remove `babel-plugin-transform-require-ignore` line in `.babelrc` and Babel will crash testing:

```
Critical error:
SyntaxError: /Users/teryaew/Projects/common/gemini-react-example/src/button/button.css: Unexpected token (1:0)
> 1 | .button {
    | ^
```

Also there is issue with importing `React` for every gemini test or using `babel-plugin-react-require`. You can reproduce it in repo too. Maybe there some solution other than that?

P.S.: My `node -v` is `v6.5.0`.

**Comment by SevInf** [comment]
Hi.
We do not use non-js loaders internally yet and I don't have the ready answer regarding CSS. But this is definitely something we should support. 
Probably, we need to integrate something like [webpack-isomorphic-tools](https://github.com/halt-hammerzeit/webpack-isomorphic-tools) or [universal-webpack](https://github.com/halt-hammerzeit/universal-webpack) into [WebpackBundler](https://github.com/researchgate/gemini-react/blob/master/lib/webpack-bundler.js).

Unfortunately, I won't have time to look into it at least until the end of next week.

**Comment by SevInf** [comment]
Still haven't got to it, but right come up with a workaround. In node environment, we don't actually need proper results from loaders, we just need it to execute it without exceptions. So, doing something like this:

``` js
require.extensions['css'] = () => ({});
```

should be enough to fix it.

**Comment by tx44** [comment]
How about to ignore breaking require extensions by including `ignore-styles` module in `setup.js` (#20) ?

**Comment by SevInf** [comment]
Fixed in 0.7.2
